debian/control

```diff
@@ -7,9 +7,8 @@ Standards-Version: 3.9.4
 
 Package: google-cloud-ops-agent
 Architecture: any
-Depends: rsyslog | system-log-daemon,
-         ${shlibs:Depends},
-         ${misc:Depends}
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Recommends: rsyslog | system-log-daemon
```

Comment:
There was discussion about swapping the order of these to put the virtual package `system-log-daemon` first. I found [these docs from Debian](https://www.debian.org/doc/debian-policy/ch-relationships.html#virtual-packages-provides) that suggest to use this ordering (virtual package second).

Reply:
Are you referring to the following paragraph:
> To specify which of a set of real packages should be the default to satisfy a particular dependency on a virtual package, list the real package as an alternative before the virtual one.

? Any reason to not just use the virtual package as the dependency?

Reply:
I am referring to that paragraph.

If we just specify a virtual package, how will it know which one to install? I was under the impression we had to give it at least one real package to specify with the virtual as an alternative.

Reply:
Makes sense.